common/src/main/java/org/apache/celeborn/common/network/protocol/OpenStream.java

```diff
@@ -24,25 +24,13 @@
 import io.netty.buffer.ByteBuf;
 
 /** Request to read a set of blocks. Returns {@link StreamHandle}. */
-public final class OpenStream extends RequestMessage {
+public class OpenStream extends RequestMessage {
   public byte[] shuffleKey;
   public byte[] fileName;
-  public int startMapIndex;
-  public int endMapIndex;
 
-  public OpenStream(String shuffleKey, String fileName, int startMapIndex, int endMapIndex) {
-    this(
-        shuffleKey.getBytes(StandardCharsets.UTF_8),
-        fileName.getBytes(StandardCharsets.UTF_8),
-        startMapIndex,
-        endMapIndex);
-  }
-
-  public OpenStream(byte[] shuffleKey, byte[] fileName, int startMapIndex, int endMapIndex) {
+  public OpenStream(byte[] shuffleKey, byte[] fileName) {
```

Comment:
I think refactoring OpenStream will cause incompatibility issue. I think we can try renaming OpenStream to OpenChunkStream, and we should run test case with newer client and older worker to verify.